pepper-apis/src/main/java/org/broadinstitute/ddp/util/StudyDataLoaderMain.java

```diff
@@ -347,6 +347,8 @@ public static void initDbConnection() {
         JsonElement combinedConsentSurveyData = surveyData.getAsJsonObject().get("combinedconsentsurvey");
         JsonElement followupSurveyData = surveyData.getAsJsonObject().get("followupsurvey");
         JsonElement followupConsentSurveyData = surveyData.getAsJsonObject().get("followupconsentsurvey");
+        JsonElement prionConsentSurveyData = surveyData.getAsJsonObject().get("ConsentSurvey");
+        JsonElement medicalSurveyData = surveyData.getAsJsonObject().get("MedicalSurvey");
 
```

Comment:
I'm not too sure about study migration and loader code, but I hope this doesn't conflict with what we're doing for migration other studies. Please coordinate with @ssettipalli when he gets back.